icons/pbs.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>PBS_watermark_QuarterOpac</title><g style="opacity:0.25"><path d="M175.588 72.548c-1.324 0-1.917.79-1.917 1.642 0 1.001.675 1.482 1.418 1.875.744.393 1.011.557 1.011.976 0 .353-.307.557-.821.557-.715 0-1.222-.33-1.551-.595v1.205a3.1 3.1 0 0 0 1.542.474c1.285 0 2.096-.67 2.096-1.75 0-1.116-.952-1.592-1.5-1.877s-.991-.515-.991-.956c0-.311.266-.504.714-.504a2.7 2.7 0 0 1 1.368.387V72.85a3.7 3.7 0 0 0-1.369-.3m-18.503.056c-1.374 0-2.54 1.125-2.51 2.5.025 1.026.687 1.792 1.606 2.082v1.438h2.255v-1.197h.282a.63.63 0 0 0 .652-.621v-1.03l.558-.113h.003a.185.185 0 0 0 .13-.267l-1.464-2.792zm2.597 0 1.463 2.792a.185.185 0 0 1-.127.267l-.558.114v1.029a.63.63 0 0 1-.652.621h-.284v1.197h.916v-1.197h.281a.63.63 0 0 0 .653-.621v-1.03l.557-.113a.185.185 0 0 0 .132-.267l-1.463-2.792zm4.8 0v6.02h1.184V76.71h.237c1.483 0 2.404-.79 2.404-2.06.002-1.28-.836-2.046-2.238-2.046zm4.544 0v6.02h1.92c1.472 0 2.141-.945 2.141-1.822 0-.694-.38-1.23-.996-1.42.4-.278.596-.687.596-1.237 0-.95-.696-1.54-1.822-1.54zm-3.36 1.003h.348c.697 0 1.145.403 1.145 1.025 0 .676-.414 1.062-1.136 1.062h-.357zm4.546.014h.568c.464 0 .767.29.767.736 0 .447-.328.742-.83.742h-.505zm-12.233.5a.573.573 0 0 1 .751.546.574.574 0 0 1-.573.573.573.573 0 0 1-.178-1.119m12.233 1.927h.75c.598 0 .955.292.955.781 0 .524-.358.79-1.064.79l-.64.004z" style="fill:#000000;stroke-width:0.0696785" transform="translate(-154.178 -64.059)"/></g></svg>
```

Comment:
The `<title>` is incorrect here, you have extra attributes and styling, there is a `<g>` around the `<path>`, etc. Again, please read https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#2-extract-the-icon-from-the-logo and the following steps.